src/solver.jl

```diff
@@ -87,6 +94,16 @@ function PajaritoSolver(;
     tol_prim_infeas = 1e-6,
     )
 
+    if mip_solver == UnsetSolver()
+        error("No MIP solver specified (set mip_solver)\n")
```

Comment:
we can create a new issue to give more detailed error and warning messages if you like